toolchain/lc-compile/src/report.c

```diff
@@ -26,6 +26,8 @@ extern int IsDependencyCompile(void);
 
 ////////////////////////////////////////////////////////////////////////////////
 
+int ErrorInfo = 0;
```

Comment:
Most of the 'option' globals in lc-compile are of the form OutputAsC / ErrorInfo - so there is some inconsistency here. 's_*' is usually used a compilation-unit local var, so those which are shared amongst units should really be 'g_*'. I'll leave this as ErrorInfo for now, and tweak the initialization. We can clean up this naming in a different patch uniformly.